CorsixTH/Lua/config_finder.lua

```diff
@@ -476,6 +485,9 @@ audio_music = nil -- [[X:\ThemeHospital\Music]]
 'scroll_speed = ' .. tostring(config_values.scroll_speed) .. '\n' ..
 'shift_scroll_speed = ' .. tostring(config_values.shift_scroll_speed) .. '\n' .. [=[
 
+
```

Comment:
Undo this change. Not consistent with other dividers. If you want to make another commit or pr making these section headers consistent it would be appreciated but not necessary.